src/utils/EventManager.js

```diff
@@ -121,8 +121,8 @@ define(function (require, exports, module) {
      * listen to these events should register their named eventHandler with `EventManager`.
      *
      * By default, only origins part of `window.Phoenix.TRUSTED_ORIGINS` are whitelisted. If your extension is
-     * bringing in a cross-origin ifrmame say `http://mydomain.com`, you should add it to the whitelist by setting
-     * `window.Phoenix.TRUSTED_ORIGINS["http://mydomain.com"]=true;`
+     * bringing in a cross-origin ifrmame say (`http://mydomain.com`), you should add it to the whitelist by setting
+     * `window.Phoenix.TRUSTED_ORIGINS ("http://mydomain.com") = true;`
```

Comment:
this cannot sadly be changed, it has to be square brackets.

Reply:
Fixed!